README.md

```diff
@@ -148,17 +148,21 @@ No PECL extensions is required. The [Go! AOP](http://go.aopphp.com/) library doe
 
 ## Requirements
 
-* `PHP 7.4`.
-* `Go! AOP 3.0`
+* `PHP 8`
+* `Go! AOP 4.x`
 
 ## Installation
 
 ### 1. Add aspect-mock to your composer.json.
 
+(lines with goaop @dev are needed depending on the `minimum-stability` in your composer settings)
+
 ```
 {
 	"require-dev": {
-		"codeception/aspect-mock": "*"
+		"codeception/aspect-mock": "*",
```

Comment:
Branch alias allows to set version constraint to `"5.*` or `5.0.*`.

I'm a bit rusty with composer, please check https://getcomposer.org/doc/articles/aliases.md

Reply:
There is no release yet, so you need at least a dev version constraint. 5.0-dev is the current alias for master.

So: `~5.0@dev` will also work, but I didn't change it in this PR since its not related to the PHP upgrade.

Let me know what it should be or feel free to edit the code directly.